autoscheduler/scraper/models/instructor.py

```diff
@@ -1,4 +1,10 @@
 from django.db import models
 
 class Instructor(models.Model):
-    """ Describe model here """
+    """ Model that represents a professor, who may teach many courses """
+    id = models.IntegerField(primary_key=True)
+    email_address = models.CharField(max_length=320, null=True)

```

Comment:
Max length can be around 48, querying all of the instructors for 2019(in all departments) didn't have anyone with an email longer than that